tests/Endpoints/TasksTest.php

```
@@ -48,6 +48,16 @@ public function testGetOneTaskClient(): void
         self::assertInstanceOf(DocumentAdditionOrUpdateDetails::class, $task->getDetails());
     }
 
+    public function testGetTaskDocumentsClient(): void
+    {
+        [$seedTask, $completedTask] = $this->seedIndex();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$seedTask'. (undefined)

(UnusedLocalVariable)